spec/rubocop/cop/ezcater/rspec_require_http_status_matcher_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe RuboCop::Cop::Ezcater::RspecRequireHttpStatusMatcher do
+  subject(:cop) { described_class.new }
+
+  it "accepts usage of the HTTP status matcher" do
+    inspect_source("expect(response).to have_http_status :bad_request")
+    expect(cop.offenses).to be_empty
+  end
+
+  it "registeres an offence when attempting to assert directly on the status" do
+    inspect_source("expect(response.code).to eq 400")
+    expect(cop.messages).to match_array(["Use the `have_http_status` matcher, like "\
+      "`expect(response).to have_http_status :bad_request`, rather than `expect(response.code).to eq 400`",])
+  end
+end
```

Comment:
Should there be a test case for `expect(response).to be_bad_request`?

Reply:
I wasn't planning on banning that style since it seemed non-trivial and, while not our preferred syntax, still provides more documentation value than asserting around magic integers.

Reply:
@bjackson2 and @mjfaga - do y'all think it's worth banning the `expect(response).to be_bad_request` style, too?

Reply:
I'm OK with the `be_foo` style too, we use that style in a buncha places

Reply:
I'm fine with leaving that in place. I think the goal/scope here is to avoid explicit status codes (`400`, `401`) in our specs and write more readable specs (`:bad_request`, `:unauthorized`).